documentloaders/pdf_test.go

```diff
@@ -0,0 +1,25 @@
+package documentloaders
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPDFLoader(t *testing.T) {
+	t.Parallel()
+	file, err := os.Open("./testdata/lorem-ipsum.pdf")
+	assert.NoError(t, err)
+
+	loader := NewPDF(file)
+
+	docs, err := loader.Load(context.Background())
+	require.NoError(t, err)
+
+	expectedText := "Te quo illum phaedrum salutatus, has in quis alii vide."
+	assert.Contains(t, docs[0].PageContent, expectedText)
+	assert.Equal(t, docs[0].Metadata["Pages"], "1")
```

Comment:
You have arguments the wrong way round here. It should be: `assert.Equal(t, "1", docs[0].Metadata["Pages"])`

Reply:
fixed, I was copying test strategy from `TestCSVLoader`